### PR TITLE
samples: bluetooth: Remove small targets from supported list.

### PR DESCRIPTION
**Pull request by grochu**
Bugfix for release: remove nRF52810, nRF52811, nRF52820 from "supported list" in selected samples (.rst and sample.yaml).

### CODE REVIEW

**Review by KAGA164 — APPROVED** [review]
Looks good

File: samples/bluetooth/peripheral_gatt_dm/sample.yaml [hunk 1]
```diff
@@ -5,6 +5,6 @@ tests:
   samples.bluetooth.peripheral_gatt_dm:
     build_only: true
     build_on_all: true
-    platform_allow: nrf51dk_nrf51422 nrf52dk_nrf52832 nrf52840dk_nrf52840 nrf52dk_nrf52810
+    platform_allow: nrf51dk_nrf51422 nrf52dk_nrf52832 nrf52840dk_nrf52840
```
Comment (Tschet1) [inline samples/bluetooth/peripheral_gatt_dm/sample.yaml:8]: Did you check the compilation with nrf51dk_nrf51422? Should this be marked as supported in the Readme?

File: samples/bluetooth/central_dfu_smp/sample.yaml [hunk 2]
```diff
@@ -3,12 +3,12 @@ sample:
...
   samples.bluetooth.central_dfu_smp:
-    platform_allow: nrf51dk_nrf51422 nrf52dk_nrf52832 nrf52840dk_nrf52840 nrf52dk_nrf52810
+    platform_allow: nrf51dk_nrf51422 nrf52dk_nrf52832 nrf52840dk_nrf52840
     harness: bluetooth
     tags: bluetooth
   samples.bluetooth.central_dfu_smp.build:
     build_only: true
     build_on_all: true
-    platform_allow: nrf51dk_nrf51422 nrf52dk_nrf52832 nrf52840dk_nrf52840 nrf52dk_nrf52810
+    platform_allow: nrf51dk_nrf51422 nrf52dk_nrf52832 nrf52840dk_nrf52840
```
Comment (Tschet1) [inline samples/bluetooth/central_dfu_smp/sample.yaml:12]: Should this line be set to the same platforms that we specify in the Readme? But maybe I just fully understand what this file is used for.

File: samples/bluetooth/peripheral_gatt_dm/sample.yaml (hunk 1 above)
Comment (Tschet1) [inline samples/bluetooth/peripheral_gatt_dm/sample.yaml:8]: Did you check the compilation with nrf51dk_nrf51422? Should this be marked as supported in the Readme?

**Review by Tschet1 — COMMENTED** [review]
Did you also double-check the peripheral_uart sample with the SDC?

File: samples/bluetooth/central_dfu_smp/sample.yaml (hunk 2 above)
Comment (peknis) [reply samples/bluetooth/central_dfu_smp/sample.yaml:12]: Just wondering the same as Jan. Should the list in yaml match the table in the README? Now, they don't match.

Comment (grochu) [reply samples/bluetooth/central_dfu_smp/sample.yaml:12]: For now, this information in the yaml file are just used for deciding which platforms should be used for testing, but don't know it is still used. yaml refactoring will be done as part of: https://projecttools.nordicsemi.no/jira/browse/NCSDK-8508

File: samples/bluetooth/peripheral_gatt_dm/sample.yaml (hunk 1 above)
Comment (grochu) [reply samples/bluetooth/peripheral_gatt_dm/sample.yaml:8]: I think the overall approach is to not have the nRF51 mentioned in the documentation in NCS, so I wouldn't change this. This info in yaml might be outdated, but is subject to clean up with https://projecttools.nordicsemi.no/jira/browse/NCSDK-8508

**Review by peknis — APPROVED** [review]

**Comment by grochu** [comment]
> Did you also double-check the peripheral_uart sample with the SDC?

I checked nRf52810, nRF52820 and nRF52811 with SDC but only with the prj_minimal.conf - which is dedicated to these small targets. Then I think we should be safe saying these targets are supported.

File: samples/bluetooth/central_dfu_smp/sample.yaml (hunk 2 above)
Comment (Tschet1) [reply samples/bluetooth/central_dfu_smp/sample.yaml:12]: I see. Probably fine then. It would be nice to test at least the building of the samples in CI.

**Comment by Tschet1** [comment]
> > Did you also double-check the peripheral_uart sample with the SDC?
> 
> I checked nRf52810, nRF52820 and nRF52811 with SDC but only with the prj_minimal.conf - which is dedicated to these small targets. Then I think we should be safe saying these targets are supported.

Should we clearify this in the README?

**Comment by grochu** [comment]
> Should we clearify this in the README? I.e. for which kits you need to use the minimal config?

Yes, such clarification can help. Let me fix this.

**Comment by grochu** [comment]
@peknis, @Tschet1, @KAGA164  please kindly check the additional update in the _samples/peripheral_uart/README.rst_.

File: samples/bluetooth/peripheral_uart/README.rst [hunk 3]
```diff
@@ -41,6 +41,9 @@ The sample supports the following development kits:
    :rows: nrf5340dk_nrf5340_cpuapp_and_cpuappns, nrf52840dk_nrf52840, nrf52840dk_nrf52811, nrf52833dk_nrf52833, nrf52833dk_nrf52820, nrf52833dk_nrf52820, nrf52dk_nrf52832, nrf52dk_nrf52810
 
 
+.. note::
+   To build this sample for ``nrf52dk_nrf52810``, ``nrf52840dk_nrf52811`` or ``nrf52833dk_nrf52820``. use the `Minimal build`_ approach.
```
Comment (peknis) [inline samples/bluetooth/peripheral_uart/README.rst:45]:
```suggestion
   To build this sample for ``nrf52dk_nrf52810``, ``nrf52840dk_nrf52811`` or ``nrf52833dk_nrf52820``, use the `Minimal build`_ approach.
```

**Review by peknis — CHANGES_REQUESTED** [review]

**Review by peknis — APPROVED** [review]

**Comment by Tschet1** [comment]
thanks. looks good.

**Review by Tschet1 — APPROVED** [review]

**Pull request merged**